Apps/W1/ReviewGLEntries/app/permissions/ReviewGLEntriesRead.PermissionSet.al

```diff
@@ -12,6 +12,6 @@ permissionset 22219 "Review G/L Entries - Read"
 
     IncludedPermissionSets = "Review G/L Entries - Objects";
 
-    Permissions = tabledata "G/L Entry Review Entry" = r,
+    Permissions = tabledata "G/L Entry Review Log" = r,
```

Comment:
What about the old permissions? Are they no longer needed? Should they not stay (but framed in CLEAN tags?

...I've added them back in the internal PR, as I don't think we just can remove the permissions, before the table is removed.